app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -1,451 +1,464 @@
-                                    R.string.invalid_case_property_length,
-                                    e.getCaseProperty()));
-                } catch (InvalidStructureException e) {
-                    // Record will be wiped when form entry is exited
-                    Logger.log(LogTypes.TYPE_ERROR_WORKFLOW, e.getMessage());
-                    throw new IllegalStateException(e.getMessage());
-                } catch (Exception e) {
-                    NotificationMessage message =
-                            NotificationMessageFactory.message(NotificationMessageFactory.StockMessages.FormEntry_Save_Error,
-                                    new String[]{null, null, e.getMessage()});
-                    CommCareApplication.notificationManager().reportNotificationMessage(message);
-                    Logger.log(LogTypes.TYPE_ERROR_WORKFLOW,
-                            "Error processing form. Should be recaptured during async processing: " + e.getMessage());
-                    throw new RuntimeException(e);
-                } finally {
-                    userDb.endTransaction();
-                }
-            }
-        }
-    }
-
-    /**
-     * Update the form record
-     *
-     * @return The updated form record, which has been written to storage.
-     */
-    private FormRecord updateAndWriteRecord()
-            throws InvalidStateException {
-        try {
-            return FormRecordCleanupTask.updateAndWriteRecord(CommCareApplication.instance(),
-                    this, CommCareApplication.instance().getUserStorage(FormRecord.class));
-        } catch (InvalidStructureException e1) {
-            e1.printStackTrace();
-            throw new InvalidStateException("Invalid data structure found while parsing form. There's something wrong with the application structure, please contact your supervisor.");
-        } catch (XmlPullParserException | IOException e) {
-            e.printStackTrace();
-            throw new InvalidStateException("There was a problem with the local storage and the form could not be read.");
-        } catch (UnfullfilledRequirementsException e) {
-            throw new RuntimeException(e);
-        }
-    }
-
-    private static class InvalidStateException extends Exception {
-        public InvalidStateException(String message) {
-            super(message);
-        }
-    }
-
-    public void setDisplayName(String displayName) {
-        this.displayName = displayName;
-    }
-
-    public String getDisplayName() {
-        return displayName;
-    }
-
-    public void setStatus(@FormRecordStatus String status) {
-        this.status = status;
-    }
-
-    public String getXmlns() {
-        return xmlns;
-    }
-
-    public void setUuid(String uuid) {
-        this.uuid = uuid;
-    }
-
-    public void setLastModified(Date lastModified) {
-        this.lastModified = lastModified;
-    }
-}
+package org.commcare.android.database.user.models;
```

Comment:
Out of curiosity, do you know why Github failed to diff this file?

Reply:
Maybe because I renamed the file.